code/modules/reagents/reagent_dispenser.dm

```diff
@@ -62,6 +62,13 @@
 	tank_volume = 100000
 
 
+/obj/structure/reagent_dispensers/oil
+	name = "oil tank"
+	desc = "A tank of oil, commonly used to by robotics to fix leaking IPCs or just to loosen up those rusted underused parts."
+	icon_state = "oil"
+	reagent_id = "oil"
+	tank_volume = 2000

```

Comment:
I'm tempted to say make this 3000; dunno. It's 560 units of blood to fill up a mob; I do admit I have some concern that IPCs won't be able to find oil on some shifts if they get beaten up much.

Reply:
Number is adjustable, i just used one you gave me before. Just give me a good number and i will change it to it.